### ISSUE TITLE
OCR by OneNote

### ISSUE THREAD

**Issue by MikeGTi** [jceresearch/onenote_vba_host#8]
I want to code OCR recognizer based on Your code. Because I think, now only One Note (from all MS Office programs) have OCR part.

**Comment by jceresearch** [comment]
As you probably have seen, there is very little out there in terms of documentation of the api/xml let alone examples for OneNote 
It is all very fragile and one post i saw about using ocr in OneNote suggested it is not very good , and doesn’t happen immediately, so brace for a lot of trial and error 

I started to work on this tool to solve the problem of exporting, that was my main issue as at work we needed to upload all the notes as ms word or pdf to our cms. 
Also onetastic didn’t have that feature and on top I couldn’t get installed at work any more for security reasons 
Then it seems that Microsoft doesn’t know what to do exactly with OneNote “desktop” , kind of looks like legacy but still alive ... reluctantly . 
I fear that it will be gradually left behind and eventually any vba will stop working. It is already tricky do the stuff from excel. After all vba is frozen since 2010.
OneNote online seems to be still alive (they update the app regularly) but it is still the “poor cousin” compared with the rest ... no alerting /comments with @ and I am finding it tricker to justify using OneNote for team collaboration when there are wikis , lists, planner/tasks and word documents in Teams all very much integrated. 

That is why I don’t invest much time on this and I would recommend you explore other options. there seems to be  ways of batch extracting images from OneNote ... maybe 
 then using other tools to do ocr? or just using other tools for your use case end to end?
Possibly google docs or some other online service may be better to do ocr, or perhaps a nice python library , saw google api seems to be free for first 1000 pages each month .. (just eyeballing , I haven’t really checked) 

https://source.opennews.org/articles/so-many-ocr-options/

My 2p :)

Best of luck in any case
